nio.conf

```diff
@@ -166,6 +166,10 @@ secure=[[PK_SECURE]]
 brews=pubkeeper.brew.local.brew.LocalBrew,pubkeeper.brew.websocket.brew.WebsocketBrew
 
 [pubkeeper_server]
+# Specifies whether this nio instance launches its own Pubkeeper Server
+#
+#standalone=False
```

Comment:
I'm not sure how we'd do it, but it would be cool if we could set this to `True` by default for some binaries and `False` by default for others. Any ideas how we could do that?

Reply:
At this point, I don't think binaries know their name.  What we could do, is have an internal env-variable (?) that we inject when building a binary, or rather, when building a binary sed-replace some variable inside.

Reply:
Something in the binary has to do it, only the first time, and to the right project, seems like the number of conditions to meet keeps increasing...